src/com/intellij/idea/plugin/hybris/common/utils/HybrisIcons.java

```diff
@@ -50,6 +50,7 @@ public interface HybrisIcons {
     Icon TYPE_SYSTEM = getIcon("/icons/typeSystem.svg");
     Icon BEAN = getIcon("/icons/beanIcon.svg");
     Icon LOCALIZED = getIcon("/icons/localized.svg");
+    Icon INTERCEPTOR = getIcon("/icons/interceptor.svg");
```

Comment:
in case of "dark" theme SVG icon will not be recognisable 
<img width="116" alt="image" src="https://user-images.githubusercontent.com/2292510/197382736-08d4a018-b29c-4390-b67f-9f25d17061a6.png">